packages/dds/tree/src/util/offsetList.ts

```diff
@@ -8,12 +8,23 @@ export type OffsetList<TContent = Exclude<unknown, number>, TOffset = number> =
 /**
  * Helper class for constructing an offset list that...
  *  - Does not insert offsets if there is no content after them
- *  - Merges runs offsets together
+ *  - Does not insert 0-sized offsets
+ *  - Merges runs of offsets together
  */
 export class OffsetListFactory<TContent> {
 	private offset = 0;
 	public readonly list: OffsetList<TContent> = [];
 
+	public push(...offsetOrContent: (number | TContent)[]): void {
```

Comment:
It looks like this file has different indentation than the other files in the package.

Reply:
Fixed